crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -4636,6 +4660,11 @@ fn test_eligible_for_ff_output_coin(#[case] amount: u64, #[case] ph: &str, #[cas
 #[rstest]
 #[case(ASSERT_MY_PARENT_ID, "{h1}")]
 #[case(ASSERT_MY_COIN_ID, "{coin12}")]
+#[case(ASSERT_MY_AMOUNT, "123")]
```

Comment:
it's not obvious why this fails. we allow this condition as the first one, right?
I have a feeling that maybe this test isn't testing what we think it is.

Reply:
this test was not what we thought it was - it was actually testing conditions order